### PR TITLE
Initial template migration

### PR DESCRIPTION
**Pull request by the-scrambler**

### CODE REVIEW

File: app/src/main/res/drawable/ic_launcher_background.xml [hunk 1]
```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="utf-8"?>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (yenerm) [inline app/src/main/res/drawable/ic_launcher_background.xml:2]: missing license

File: app/src/main/res/drawable-v24/ic_launcher_foreground.xml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (yenerm) [inline app/src/main/res/drawable-v24/ic_launcher_foreground.xml:1]: missing license

File: app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
```
Comment (yenerm) [inline app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml:2]: missing license

File: app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
```
Comment (yenerm) [inline app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml:2]: missing license

File: settings.gradle [hunk 5]
```diff
@@ -0,0 +1,16 @@
+pluginManagement {
```
Comment (yenerm) [inline settings.gradle:1]: missing license

**Review by yenerm — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 6]
```diff
@@ -0,0 +1,34 @@
+*/.gitignore
```
Comment (kkuan2011) [inline .gitignore:1]: same comment here to add some spacing between the sections per this example: https://github.com/android/compose-samples/find/main

**Pull request closed without merging**